### ISSUE TITLE
keras model predict the image and accuracy was very low

### ISSUE THREAD

**Issue by Npccc** [jhfjhfj1/autokeras#271]
I used the keras model output from autokeras to predict the image and found that the accuracy was very low。
---------------------------------------------
in train.py ImageClassifier.evaluate()   accuracy:94%
but use the keras model output from autokeras to predict the image    accuracy:21%

code:
model = load_model("classification_5.h5")
x = model.output
x = Activation('softmax', name='activation_add')(x)
model = Model(model.input, x)

testImg = img_to_array(img)
testImg = testImg.astype('float32') / 255
testImg = np.reshape(testImg, (1, IMAGE_SIZE, IMAGE_SIZE, 3))
pred = model.predict(testImg)
predict = np.argmax(pred, axis=1)

**Comment by jhfjhfj1** [comment]
@Npccc Please refer to this issue #186 .
We will try to make it easier to use in the future. Thanks.
